stories/documentation/feedback/empty-state/html&css/empty-state-page.stories.scss

```diff
@@ -1,4 +1,3 @@
 :host {
 	display: block;
-	height: 100vh;
```

Comment:
Doesn't this make the story very small?

Reply:
I've added a min-height in another PR to deal with this.